3p/integration.js

```diff
@@ -200,14 +200,6 @@ const defaultAllowedTypesInCustomFrame = [
   '_ping_',
 ];
 
-// List of ad networks that will manually call `window.context.renderStart` to
-// emit render-start event when ad actually starts rendering. Please add
-// yourself here if you'd like to do so (which we encourage).
-export const waitForRenderStart = [
```

Comment:
We used to use it in `integration.js`. But now we decide to always send `bootstrap-loaded` (used to be `render-start`) message, and let `amp-ad` decide whether to listen to it or not.